R/util.r

```diff
@@ -7,7 +7,7 @@
 #' @keywords internal
 # @param short  Whether to use unqualified host name (before first dot)
 host = function(node=getOption("clustermq.host", Sys.info()["nodename"]),
-                ports=6000:9999, n=100) {
+                ports=getOption("clustermq.port.range", 6000:9999), n=100) {
```

Comment:
Please name the option `clustermq.port` to be consistent with the ssh option and document that it can take a range

Reply:
Thanks - now using `clustermq.port` and also updated documentation accordingly.